### PR TITLE
Dev/pg v13

### PR DESCRIPTION
**Pull request by danperezsan**
- Update golang:1.15-alpine3.12 to avoid build failures on latest version: https://gitlab.alpinelinux.org/alpine/aports/-/issues/12396
- Update versioning documentation

### CODE REVIEW

**Review by gbmeuk — DISMISSED** [review]

File: scripts/setup-master.sh [hunk 1]
```diff
@@ -30,8 +30,14 @@ echo "Adding replication specific configuration"
...
+  echo "max_wal_senders = 10"
+  echo "wal_keep_size = 2048"
   echo "hot_standby = on"
-  echo "synchronous_standby_names = '*'"
 } >>"$PGDATA"/postgresql.conf
+
+if [[ ${SYNC_REPLICATION^^} == TRUE ]]; then
+  echo "Adding synchronous replication specific configuration"
+  {
+    echo "synchronous_standby_names = '*'"
```
Comment (gbmeuk) [inline scripts/setup-master.sh:41]: There's no async equivalent of this?

Comment (danperezsan) [reply scripts/setup-master.sh:41]: Nope. Replication is asynchronous by default so there is really no need to be explicit about it.
https://www.postgresql.org/docs/13/warm-standby.html#SYNCHRONOUS-REPLICATION

Comment (gbmeuk) [reply scripts/setup-master.sh:41]: So I assume that it's just a fire and forget, there's no ACL?

Comment (danperezsan) [reply scripts/setup-master.sh:41]: It's just that.

**Review by gbmeuk — APPROVED** [review]

**Pull request merged**